microsoft-edge/web-platform/release-notes/133.md

```diff
@@ -0,0 +1,299 @@
+
+###### Animation.overallProgress
+
+Adds an `overallProgress` property to the JavaScript class [Animation](https://developer.mozilla.org/docs/Web/API/Animation).
+
+This property provides authors with a convenient and consistent representation of how far along an animation has advanced across its iterations and regardless of the nature of its [timeline](https://developer.mozilla.org/docs/Web/API/AnimationTimeline).
+
+###### Atomics.pause
+
+Adds the `Atomics.pause` method to hint the CPU that the current code is executing a spinlock.
+
+###### CSP hash reporting for scripts
+
+Complex web application often need to keep tabs of the subresources that they download, for security purposes.
+
+In particular, upcoming industry standards and best practices (e.g. PCI-DSS v4) require that web applications keep an inventory of all the scripts they download and execute.
+
+This feature builds on CSP and the Reporting API to report the URLs and hashes (for CORS/same-origin) of all the script resources that the document loads.
+
+###### CSS :open pseudo-class
+
+The `:open` pseudo-class matches `<dialog>` and `<details>` when they are in their open state, and matches `<select>` and `<input>` when they are in modes which have a picker and the picker is showing.
+
+###### CSS Scroll State Container Queries
+
+Use container queries to style descendants of containers based on their scroll state.
+
+The query container is either a scroll container, or an element affected by the scroll position of a scroll container. The following states can be queried:
+
+* Whether a sticky positioned container is stuck to one of the edges of the scroll box (stuck)
+* Whether a scroll snap aligned container is currently snapped horizontally or vertically (snapped)
+* Whether a scroll container can be scrolled in a queried direction (scrollable)
+
+A new `container-type:scroll-state` is introduced to allow such containers to be queried.
+
+For instance:
+
+```css
+#sticky {
+  position: sticky;
+  container-type: scroll-state;
+}
+
+@container scroll-state(stuck: top) {
+  #sticky-child { font-size: 75% }
+}
+```
+
+###### CSS advanced attr() function
+
+Implement the augmentation to `attr()` specified in CSS Level 5, which allows types besides `<string>` and usage in all CSS properties (besides pseudo-element `content`).
+
+Example:
+
+```html
+<style>
+  div {
+     background-color: attr(data-foo type(<color>), red);
+  }
+</style>
+<div data-foo="blue">test</div>
+```
+
+###### CSS text-box, text-box-trim, and text-box-edge
+
+To achieve optical balance of text content, the `text-box-trim` and `text-box-edge` properties, along with the `text-box` shorthand property, make finer control of vertical alignment of text possible.
+
+The `text-box-trim` property specifies the sides to trim, above or below, and the `text-box-edge` property specifies how the edge should be trimmed.
+
+These properties allow developers to control vertical spacing precisely by using the font metrics.
```

Comment:
```suggestion
These properties allow you to control vertical spacing precisely by using the font metrics.
```
https://learn.microsoft.com/en-us/style-guide/grammar/person#in-general-use-second-person-pronouns-you-your